Barotrauma/BarotraumaClient/ClientSource/GUI/GUITextBox.cs

```diff
@@ -617,10 +617,15 @@ public void ReceiveCommandInput(char command)
                 case '\b' when !Readonly: //backspace
                 {
                     using var _ = new TextPosPreservation(this);
-                    if (PlayerInput.KeyDown(Keys.LeftControl) || PlayerInput.KeyDown(Keys.RightControl))
+                    if (PlayerInput.IsCtrlDown())
                     {
-                        SetText(string.Empty, false);
-                        CaretIndex = Text.Length;
+                        var removeTo = FindStopSymbolLeft(Text, CaretIndex);
```

Comment:
With simple types like `int` and `string,` using `var` just adds mental payload for the reader without any benefits. `var` can improve readability when used with generic types (e.g. `var list = new List<MyType>()`), but shouldn't be used when it's not clear from the context what the type is.